milli/src/update/index_documents/mod.rs

```diff
@@ -1651,4 +1652,60 @@ mod tests {
th a thud.\nHEAD GUARD\nSeize him! (Donkey takes of running.) \nAfter him!\nGUARDS\nHe's getting away! Get him! This way! \nTurn!\nDonkey keeps running and he eventually runs into Shrek. Literally. \nShrek turns around to see who bumped into him. Donkey looks scared \nfor a moment then he spots the guards coming up the path. He \nquickly hides behind Shrek.\nHEAD GUARD\nYou there. Ogre!\nSHREK\nAye?\nHEAD GUARD\nBy the order of Lord Farquaad I am authorized \nto place you both under arrest and transport \nyou to a designated resettlement facility.\nSHREK\nOh, really? You and what army?\nHe looks behind the guard and the guard turns to look as well \nand we see that the other men have run off. The guard tucks tail \nand runs off. Shrek laughs and goes back about his business and \nbegins walking back to his cottage.\nDONKEY\nCan I say something to you? Listen, \nyou was really, really, really somethin' \nback here. Incredible!\nSHREK\nAre you talkin' to...(he turns around \nand Donkey is gone) me? (he turns back \naround and Donkey is right in front \nof him.) Whoa!\nDONKEY\nYes. I was talkin' to you. Can I tell \nyou that you that you was great back \nhere? Those guards! They thought they \nwas all of that. Then you showed up, \nand bam! They was trippin' over themselves \nlike babes in the woods. That really \nmade me feel good to see that.\nSHREK\nOh, that's great. Really.\nDONKEY\nMan, it's good to be free.\nSHREK\nNow, why don't you go celebrate your \nfreedom with your own friends? Hmm?\nDONKEY\nBut, uh, I don't have any friends. And \nI'm not goin' out there by myself. Hey, \nwait a minute! I got a great idea! I'll \nstick with you. You're mean, green, \nfightin' machine. Together we'll scare \nthe spit out of anybody that crosses \nus.\nShrek turns and regards Donkey for a moment before roaring very \nloudly.\r\nhttps://bug.example.com/meilisearch/milli.saml2?ROLE=Programmer-1337&SAMLRequest=Cy1ytcZT1Po%2L2IY2y9Unru8rgnW4qWfPiI0EpT7P8xjJV8PeQikRL%2E8D9A4pj9tmbymbQCQwGmGjPMK7qwXFPX4DH52JO2b7n6TXjuR7zkIFuYdzdY2rwRNBPgCL7ihclEm9zyIjKZQ%2JTqiwfXxWjnI0KEYQYHdwd6Q%2Fx%28BDLNsvmL54CCY2F4RWeRs4eqWfn%2EHqxlhreFzax4AiQ2tgOtV5thOaaWqrhZD%2Py70nuyZWNTKwciGI43AoHg6PThANsQ5rAY5amzN%2ufbs1swETUXlLZuOut5YGpYPZfY6STJWNp4QYSUOUXBZpdElYsH7UHZ7VhJycgyt%28aTK0GW6GbKne2tJM0hgSczOqndg6RFa9WsnSBi4zMcaEfYur4WlSsHDYInF9ROousKqVMZ6H8%2gbUissaLh1eXRGo8KEJbyEHbhVVKGD%28kx4cfKjx9fT3pkeDTdvDrVn25jIzi9wHyt9l1lWc8ICnCvXCVUPP%2BjBG4wILR29gMV9Ux2QOieQm2%2Fycybhr8sBGCl30mHC7blvWt%2T3mrCHQoS3VK49PZNPqBZO9C7vOjOWoszNkJx4QckWV%2FZFvbpzUUkiBiehr9F%2FvQSxz9lzv68GwbTu9fr638p%2FQM%3D&RelayState=https%3A%2F%example.bug.com%2Fde&SigAlg=http%3A%2F%2Fwww.w3.org%2F2000%2F09%2Fxmldsig%23rsa-sha1&Signature=AZFpkhFFII7PodiewTovaGnLQKUVZp0qOCCcBIUkJ6P5by3lE3Lldj9pKaFu4wz4j%2B015HEhDvF0LlAmwwES85vdGh%2FpD%2cIQPRUEjdCbQkQDd3dy1mMXbpXxSe4QYcv9Ni7tqNTQxekpO1gE7rtg6zC66EU55uM9aj9abGQ034Vly%2F6IJ08bvAq%2B%2FB9KruLstuiNWnlXTfNGsOxGLK7%2BXr94LTkat8m%2FMan6Qr95%2KeR5TmmqaQIE4N9H6o4TopT7mXr5CF2Z3\n";
```

Comment:
Could we make this smaller in order to test only what we want to test?

Reply:
Reduced!